cmd/ebrelayer/relayer/ethereum.go

```diff
@@ -0,0 +1,122 @@
+package relayer
+
+// -----------------------------------------------------
+//      Ethereum relayer
+//
+//      Initializes the relayer service, which parses,
+//      encodes, and packages named events on an Ethereum
+//      Smart Contract for validator's to sign and send
+//      to the Cosmos bridge.
+// -----------------------------------------------------
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	amino "github.com/tendermint/go-amino"
```

Comment:
this should be "github.com/cosmos/cosmos-sdk/codec"

Reply:
https://github.com/cosmos/peggy/pull/77/commits/be4c43930db5f73c577d2a55782bf464405dd549